### PR TITLE
LPS-194923 | Test Fix | Master

### PR DESCRIPTION
**Pull request by victorhcunha**
[LPS-194923](https://liferay.atlassian.net/browse/LPS-194923)

### CODE REVIEW

**Comment by victorhcunha** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 1]
```diff
@@ -1327,6 +1327,8 @@ definition {
 				fieldType = "String",
 				isRequired = "true",
 				objectName = "CustomObject");
+
```
Comment (JhossephG) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:1330]: Rename this task to "Given a published object with a field" using a commit called Wordsmith

**Review by JhossephG — CHANGES_REQUESTED** [review]
That's good, you are fixing the cause, just rename the task that precedes this block.

**Review by JhossephG — APPROVED** [review]

**Comment by JhossephG** [comment]
LGTM! @timpak it's in your hands

**Comment by andreefarias** [comment]
Hey @JhossephG,

I'll send this PR since this is a simple refactor to fix a testcase. Thanks for the review!

LGTM! 🎉

**Comment by andreefarias** [comment]
ci:forward

**Pull request closed without merging**